.github/workflows/cross-functional.yml

```diff
@@ -1,54 +0,0 @@
-# This workflow will build a Java project with Maven
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
-
-name: Cross functional automation
```

Comment:
@titusfortner can we leave this one please, I did a workshop on this for customers and also need to do a recording to show staged test execution

Reply:
Sure, maybe we can rename it or add comments since it isn't a test action for the suite, but an example of its own. :)